### DISCUSSION TITLE
Error message on [WiFiDrv] Dig-Octa

### DISCUSSION THREAD

**Discussion by marcobrianza** [forkineye/ESPixelStick#753] [Q&A]
Hello I'm working on the latest CI relese

I see a suspect message on the ethernet port in bold below, is it something to worry about?

config is here
[ESPixelStick-5C695A1B5AE0-etherror.json](https://github.com/forkineye/ESPixelStick/files/14612999/ESPixelStick-5C695A1B5AE0-etherror.json)

[WiFiDrv] WiFi SOFTAP: IP Address: '0.0.0.0
[EthDrv] Entering State: Powering Up
**[ 12719][E][esp32-hal-gpio.c:102] __pinMode(): Invalid pin selected
E (12713) gpio: gpio_set_level(227): GPIO output gpio_num error**
[EthDrv] Entering State: Connecting
[EthDrv] ETH IP Before Start: 0.0.0.0
[EthDrv] Ethernet Connecting as esps-5C695A1B5AE0
[EthDrv] Connecting to Ethernet with Static IP
[EthDrv] Entering State: Wait for IP Address
[EthDrv] Entering State: Got IP
[Artnet] Subscribed to broadcast
[Artnet] Listening for 300 channels from Universe 1 to 1
[WebMgr] Web server listening on port 80
[  FPPD] FPPDiscovery subscribed to broadcast messages on port: 32320
[  FPPD] FPPDiscovery subscribed to multicast: 239.70.80.80:32320
[EthDrv] Ethernet Connected with IP: 192.168.72.90

**Comment by MartinMueller2003** [comment]
Please put the ui in advanced mode and report the GPIO settings here.

**Reply by marcobrianza** [reply]
I'm keeping the default values (just set 2000 in the delay to try to mitigate the DHCP issue)

![screencapture-192-168-72-90-2024-03-15-12_53_51](https://github.com/forkineye/ESPixelStick/assets/1008278/4f3170a3-2339-4cd5-8ebb-3920cbf49782)

**Comment by MartinMueller2003** [comment]
Sorry. I fell off of this conversation. Has this issue been resolved?
